Makefile

```diff
@@ -1078,8 +1078,18 @@ test-debug:
 	env LIBSTORAGE_DEBUG=true $(MAKE) test
 
 test-rbd:
-	DRIVERS=rbd $(MAKE) deps
-	DRIVERS=rbd $(MAKE) ./drivers/storage/rbd/tests/rbd.test
+	env DRIVERS=rbd $(MAKE) deps
```

Comment:
Hi @codenrhoden,

The difference is that I learned more about the `env` command as I worked on this project and only used it at the beginning out of not fully understanding the difference. I rarely use the `test-debug` target and so have never changed it.